src/Redirection/Database/Redirects.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace SlimSEO\Redirection\Database;
+
+class Redirects {
+	protected $redirects;
+
+	public function __construct() {
+		$this->redirects = get_option( SLIM_SEO_REDIRECTION_REDIRECTS_OPTION_NAME );
+		$this->redirects = ! empty( $this->redirects ) ? $this->redirects : [];
+	}
+
+	public function list() : array {
+		return $this->redirects;
+	}
+
+	public function is_exists( string $url ) : bool {
+		$home_url = untrailingslashit( home_url() );
+		$url      = html_entity_decode( str_replace( $home_url, '', sanitize_text_field( wp_unslash( $url ) ) ) );
```

Comment:
Tách đoạn code xử lý URL thành 1 hàm `normalize_url`, sau này cần thì update một chỗ. Hàm này cũng dùng được ở trong method `update` nữa.

Reply:
done

Reply:
Ý a chỗ này là bao gồm cả phần loại bỏ `home_url()` trong `$url` đó, để chuyển tất cả `$url` về 1 dạng `/path/to/url` (nên để ko có trailing slash cho dễ).

Reply:
e chỉnh rồi a ạ